packages/app-project/src/components/ProjectHeader/components/ProjectTitle/ProjectTitle.js

```diff
@@ -29,13 +30,12 @@ function ProjectTitle({
   title = ''
 }) {
   const router = useRouter()
-  const owner = router?.query?.owner
-  const project = router?.query?.project
+  const { slug } = useStores()
   const linkProps = {
-    href: addQueryParams(`/${owner}/${project}`)
+    href: addQueryParams(`/${slug}`)
   }
 
-  const isCurrentPage = router?.pathname === linkProps.href
+  const isCurrentPage = router?.isReady && router?.asPath === linkProps.href
```

Comment:
`pathName` is different during SSR and once loaded in the browser. This listens for the router to be ready in the browser before checking the `asPath`.

Reply:
Links without `href` attributes aren't announced by screen readers, so you'll need to fix that in this component too.